examples/02-plot/movie.py

```diff
@@ -42,6 +42,3 @@
 
 # Be sure to close the plotter when finished
 plotter.close()
-
-###############################################################################
-# .. figure:: ../../images/auto-generated/sphere-shrinking.mp4
```

Comment:
This is not working.
https://dev.pyvista.org/examples/02-plot/movie.html
![image](https://user-images.githubusercontent.com/7513610/115155410-d9995d00-a0ba-11eb-9826-cef560570f11.png)

Reply:
Just made a comment that the movie won't render on a webpage.